src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -36,6 +36,12 @@ public sealed class DefaultHttpContext : HttpContext
         private DefaultConnectionInfo? _connection;
         private DefaultWebSocketManager? _websockets;
 
+        // This is field exists to make analyzing memory dumps easier.
+        // https://github.com/dotnet/aspnetcore/issues/29709
+#pragma warning disable CS0414
+        private bool _active;
```

Comment:
nit: Make the flag internal and add a unit test?

Should get rid of the warning too 🤓